client/ios/WRFall2019.xcodeproj/xcshareddata/xcschemes/WRFall2019-tvOS.xcscheme

```diff
@@ -55,6 +55,15 @@
       selectedDebuggerIdentifier = "Xcode.DebuggerFoundation.Debugger.LLDB"
       selectedLauncherIdentifier = "Xcode.DebuggerFoundation.Launcher.LLDB"
       shouldUseLaunchSchemeArgsEnv = "YES">
+      <MacroExpansion>
+         <BuildableReference
+            BuildableIdentifier = "primary"
+            BlueprintIdentifier = "2D02E47A1E0B4A5D006451C7"
+            BuildableName = "WRFall2019-tvOS.app"
+            BlueprintName = "WRFall2019-tvOS"
+            ReferencedContainer = "container:WRFall2019.xcodeproj">
+         </BuildableReference>
+      </MacroExpansion>
```

Comment:
did you add them manually?

Reply:
no

Reply:
I added app icons through xcode manually